README.md

```diff
@@ -15,6 +15,20 @@ Oncology extension package for ADaM in R Asset Library `{admiral}`
 
 To provide a complementary (to `{admiral}`) toolbox that enables to develop oncology disease area specifics.
```

Comment:
Does this sentence read properly:
To provide a complementary (to `{admiral}`) toolbox that enables to develop oncology disease area specifics.

Reply:
probably could be worded bit nicer - i'll update it now